### DISCUSSION TITLE
Machine learning

### DISCUSSION THREAD

**Discussion by mateogianolio** [mateogianolio/vectorious#210] [Ideas]
Continuation of #45 

> Discussion regarding the use of **Vectorious** for [machine learning](https://en.wikipedia.org/wiki/Machine_learning).

**Topics**
- [Supervised learning](https://en.wikipedia.org/wiki/Supervised_learning)
- [Clustering](https://en.wikipedia.org/wiki/Cluster_analysis)
- [Dimensionality reduction](https://en.wikipedia.org/wiki/Dimensionality_reduction)
- [Anomaly detection](https://en.wikipedia.org/wiki/Anomaly_detection)
- [Neural nets](https://en.wikipedia.org/wiki/Artificial_neural_network)

Found [this article](http://briandolhansky.com/blog/2014/10/30/artificial-neural-networks-matrix-form-part-5) explaining how to implement a simple neural network in matrix form, would be cool to have as an example in the future :)

**Comment by jcalvaradop777** [comment]
Best regards

I am trying to make a comparison of the incremental versions of dimensionality reduction methods. Could someone tell me where I can find the code for Incremental Locally Linear Embedding, or Incremental Multidimensional Scaling or Incremental Laplacian EigenMaps ?

Thank you.
